installer/roles/local_docker/tasks/upgrade_postgres.yml

```diff
@@ -1,43 +1,59 @@
 ---
 
+- name: Get full path of postgres data dir
+  shell: "echo {{ postgres_data_dir }}"
+  register: fq_postgres_data_dir
```

Comment:
`real_path` does not work here.  More info here: https://github.com/ansible/awx/pull/9078/files#r559794551